Userland/Services/NotificationServer/NotificationWindow.cpp

```diff
@@ -95,23 +96,21 @@ NotificationWindow::NotificationWindow(i32 client_id, const String& text, const
     widget.layout()->set_margins({ 8, 8, 8, 8 });
     widget.layout()->set_spacing(6);
 
-    if (icon.is_valid()) {
-        auto& image = widget.add<GUI::ImageWidget>();
-        image.set_bitmap(icon.bitmap());
-    }
+    m_image = &widget.add<GUI::ImageWidget>();
+    m_image->set_bitmap(icon.is_valid() ? icon.bitmap() : default_image);
```

Comment:
Will this *always* show the ladybug icon by default? That sounds like it might get old fast. I think it's perfectly fine to have a notification without an icon. Alternatively, if you really want icons, what do you think about *requiring* an icon from the caller?

The thing I want to avoid is putting uninformative "information" in a notification (which by definition is always low on screenspace).

Reply:
This was removed by hiding the image if it was not valid.